tests/backup/backup_restore_basic_test.go

```diff
@@ -895,3 +897,172 @@ var _ = Describe("{CustomResourceRestore}", func() {
+			scheduleName = fmt.Sprintf("%s-schedule-%v", BackupNamePrefix, time.Now().Unix())
+			namespaces := []string{"*"}
+			labelSelectors := make(map[string]string)
+			err = CreateScheduleBackup(scheduleName, backupClusterName, backupLocationName, backupLocationUID, namespaces,
+				labelSelectors, orgID, "", "", "", "", periodicSchedulePolicyName, periodicSchedulePolicyUid, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule name [%s]", scheduleName))
+			firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
+		})
+		// To ensure applications are deployed after a schedule backup is created
+		Step("Schedule applications", func() {
+			log.InfoD("Scheduling applications")
+			contexts = make([]*scheduler.Context, 0)
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+				appContexts := ScheduleApplications(taskName)
+				contexts = append(contexts, appContexts...)
+				for _, ctx := range appContexts {
+					ctx.ReadinessTimeout = appReadinessTimeout
+					namespace := GetAppNamespace(ctx, taskName)
+					log.InfoD("Scheduled application with namespace [%s]", namespace)
+					appNamespaces = append(appNamespaces, namespace)
+				}
+			}
+		})
+		Step("Validate applications", func() {
+			log.InfoD("Validating applications")
+			ValidateApplications(contexts)
+		})
+		Step("Verify new application namespaces inclusion in next schedule backup", func() {
+			log.InfoD("Verifying new application namespaces inclusion in next schedule backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+			allScheduleBackupNames, err := Inst().Backup.GetAllScheduleBackupNames(ctx, scheduleName, orgID)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching names of all schedule backups of schedule named [%s]", scheduleName))
+			currentScheduleBackupCount := len(allScheduleBackupNames)
+			log.InfoD("Current number of schedule backups is [%v]", currentScheduleBackupCount)
+			nextScheduleBackupOrdinal := currentScheduleBackupCount + 1
+			log.InfoD("Ordinal of the next schedule backup is [%v]", nextScheduleBackupOrdinal)
+			checkOrdinalScheduleBackupCreation := func() (interface{}, bool, error) {
+				ordinalScheduleBackupName, err := GetOrdinalScheduleBackupName(ctx, scheduleName, nextScheduleBackupOrdinal, orgID)
+				if err != nil {
+					return "", true, err
+				}
+				return ordinalScheduleBackupName, false, nil
+			}
+			log.InfoD("Waiting for 15 minutes for the next schedule backup to be triggered")
+			time.Sleep(15 * time.Minute)
+			nextScheduleBackupName, err = task.DoRetryWithTimeout(checkOrdinalScheduleBackupCreation, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second)
```

Comment:
Use the term `latest` instead of `next` everywhere

Reply:
To prevent any confusion, I intend to continue using "next".